### PR TITLE
Add pull request and commit message guidelines

### PR DESCRIPTION
**Pull request by florian-kuebler**
Internally, we decided on guidelines on pull requests
(http://go/stadia-orbit-one-commit-one-pr-policy) and
and commit messages (http://go/stadia-orbit-commit).
Those should be also exposed to public contributors.

This change, adds the essentials from the documents
mentioned above to the CONTRIBUTING.md

### CODE REVIEW

**Review by reichlfl — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -26,3 +26,31 @@ information on using pull requests.
...
+* One line, 72 characters max, but aim for 50
+* Capitalized (starts with a capital letter)
+* Doesn’t end with a period
+* Imperative style (“Fix bug” rather than “Fixed bug” or “Fixes bug”)
+
+### Commit body
+* Separated from subject line by blank line
+* Wrapped to 72 characters
+* Explain *what* and *why* rather than *how*
+* Add a `Test:` stanza describing how to test the change. Examples may be `Test: run OrbitBaseUnitTests` or `Test: start orbit, capture, save, restart orbit, load capture, see that it did not crash`.
```
Comment (pierricgimmig) [inline CONTRIBUTING.md:52]: I was under the impression that `Test:` was for tests that were actually performed before submitting the PR, giving the reviewer confidence that the change isn't breaking things. In that context, we should maybe rephrase as such:

"Add a `Test:` stanza describing how the change was tested."

WDYT?

**Review by pierricgimmig — COMMENTED** [review]
Hi Florian, thanks for this, please see minor comment.

File: CONTRIBUTING.md (hunk 1 above)
Comment (florian-kuebler) [reply CONTRIBUTING.md]: Done

**Review by pierricgimmig — APPROVED** [review]
Thanks!

**Review by ronaldfw — APPROVED** [review]
LGTM!
